doc/build/faq/connections.rst

```diff
@@ -396,9 +396,9 @@ If using a SQLite ``:memory:`` database, or a version of SQLAlchemy prior
 to version 0.7, the default connection pool is the :class:`.SingletonThreadPool`,
```

Comment:
i know this is not part of your PR, but remove the phrase " or a version of SQLAlchemy prior
to version 0.7".  That is not only more than ten years old, I dont even remember that we did things that way for sqlite back then.

Reply:
All done! 🤗